README.md

```diff
@@ -7,3 +7,30 @@ Python dependencies:
 ```bash
 pip3 install -r requirements.txt
 ```
+
+## Running unittests
+
+To run all unittests execute next command from project root folder:
+
+```bash
+python3 -m unittest discover -s tests
+```
+
+this will search for all `test*.py` files within `tests` directory
+and execute tests from those files.
+
+## Running integration tests
+
+File `integration-tests/full_drules_gen.py` is intended to generate drules
+files for all 6 themes from main Organic Maps repo. It could be used to understand
+which parts of the project are actually used by Organic Maps repo.
+
+Usage:
+
+```shell
+python full_drules_gen.py -d /path/to/organicmaps/data -o drules --txt
```

Comment:
```suggestion
cd integration-tests
python3 full_drules_gen.py -d ../../../data -o drules --txt
```

also consider making `full_drules_gen.py` executable and add a shebang

Reply:
Updated README.md

Reply:
Could you please update the actual command to `python3 full_drules_gen.py -d ../../../data -o drules --txt` too?

99% of cases the `/path/to/organicmaps/data` would be fixed as kothic is in `tools/kothic/`

Reply:
Ok, changed path